screensaver.video/addon.xml

```diff
@@ -1,13 +1,14 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="screensaver.video" name="VideoScreensaver" version="1.3.0" provider-name="robwebset">
+<addon id="screensaver.video" name="VideoScreensaver" version="1.4.0" provider-name="robwebset">
 	<requires>
 		<import addon="xbmc.python" version="2.14.0"/>
 		<import addon="script.module.simplejson" version="3.3.0"/>
 	</requires>
 	<extension point="xbmc.ui.screensaver" library="default.py"/>
-  	<extension point="xbmc.python.script" library="screensaver.py" />
-	<extension point="xbmc.service" library="service.py" start="login">
-		<provides>executable</provides>
+  	<extension point="xbmc.python.script" library="screensaver.py"/>
+	<extension point="xbmc.service" library="service.py" start="login"/>
+	<extension point="xbmc.python.pluginsource" library="plugin.py">
+		<provides>video</provides>
```

Comment:
Just because you have done that on several occasion doesnt mean that it is the correct way to do it ;)
Extension points never "depend on each other". With some clever refactoring you could put shared resources in a separate module add-on and then create separate add-ons for the different extension points. That way people can keep their video add-on list clean when trying out screensavers.